website/docs/Use-Cases/enhanced_inference.md

```diff
@@ -137,6 +137,37 @@ For local LLMs, one can spin up an endpoint using a package like [FastChat](http
 
 <!-- When only working with the chat-based models, `autogen.ChatCompletion` can be used. It also does automatic conversion from prompt to messages, if prompt is provided instead of messages. -->
 
+## Usage Summary
+
+The `OpenAIWrapper` from `autogen` tracks token counts and costs of your API calls. Use the `create()` method to initiate requests and `print_usage_summary()` to retrieve a detailed usage report, including total cost and token usage for both cached and actual requests.
+
+- `mode='both'` (default): print usage summary for all completions and non-caching completions.
+- `mode='actual'`: only print non-cached usage.
+- `mode='all'`: only print all usage (including cache).
```

Comment:
"both" and "all" are easy to confuse people.
What about "actual", "total', ["actual", "total"]

Reply:
Revised, thanks!